rust/src/rfb/parser.rs

```diff
@@ -110,46 +110,63 @@ pub struct ServerInit {
     pub height: u16,
     pub pixel_format: PixelFormat,
     pub name_length: u32,
-    pub name: Vec<u8>
+    pub name: Vec<u8>,
 }
 
+//done
```

Comment:
what is this comment?

Reply:
Did not mean to commit these comments. Will remove them now( I added this for parsers whose unit tests are there)